file-formats/susi/examples/zaff_example_susi.susi.json

```diff
@@ -0,0 +1,24 @@
+{
+    "formatVersion": "1",
+    "header": {
+        "description": "Example SUSI object",
+        "originalLanguage": "en",
+        "abapLanguageVersion": "standard"
+    },
+    "general": {
+        "applName": "Z_FEM3/Z_FEM3_GET_INFO",
+        "type": "WebService",
+        "hashName": "D2FF3796C3963799AE24F0224FCD96HS",
+        "maintenanceMode": "manual"
+    },
+    "authorizationObjects": [
+        {
+            "object": "S_USER_AGR",
+            "maintenanceStatus": "yFlag"
```

Comment:
yes:
    BEGIN OF co_maintenance_status,
      "! <p class="shorttext">No Default</p>
      "! No Default
      x_flag TYPE ty_maintenance_status VALUE 'X',
      "! <p class="shorttext">Default With Field Values</p>
      "! Default With Field Values
      y_flag TYPE ty_maintenance_status VALUE 'Y',
      "! <p class="shorttext">Default Without Field Values</p>
      "! Default Without Field Values
      v_flag TYPE ty_maintenance_status VALUE 'V',
      "! <p class="shorttext">Default Inactive</p>
      "! Default Inactive
      i_flag TYPE ty_maintenance_status VALUE 'I',
      "! <p class="shorttext">Unmaintained Object Status</p>
      "! Unmaintained Object Status
      u_flag TYPE ty_maintenance_status VALUE 'U',
      "! <p class="shorttext">Authorization Check Inactive</p>
      "! Authorization Check Inactive
      n_flag TYPE ty_maintenance_status VALUE 'N',
    END OF co_maintenance_status.

Reply:
Sorry, I think this is a misunderstanding :)

can we replace `y_value` and the other values to something more meaningful (human-readable) like `default_with_values` or just `with_Values`. This would show in the json `"maintenanceStatus": "defaultWithValues"`

Reply:
I saw you have just updated the example file. Thanks! 👍 

These changes need to be done also in the AFF ABAP type (in the interface) and the schema accordingly.